classes/fields/pick.php

```diff
@@ -2389,6 +2391,27 @@ public function is_autocomplete( $options ) {
 		return $autocomplete;
 	}
 
+	/**
+	 * Check if a field supports AJAX mode
+	 *
+	 * @param array $options Field options.
+	 *
+	 * @return bool
+	 *
+	 * @since 2.7.4
+	 */
+	public function can_ajax( $options ) {
+
+		$is_simple_tableless = in_array( $options['pick_object'], PodsForm::simple_tableless_objects(), true );
```

Comment:
Other tableless fields might extend the Pick field type class, we will want to use `pods_v( $options['type'] . '_object', $options )` instead of `$options['pick_object']`